lsp-erlang.el

```diff
@@ -31,15 +31,15 @@
   :group 'lsp-mode
   :link '(url-link "https://github.com/erlang-ls/erlang_ls"))
 
-(defcustom lsp-erlang-server-install-dir
-  "."
-  "Path to the Erlang Language Server installation dir."
+(defcustom lsp-erlang-server-path
```

Comment:
I like the change, but please update the README at https://github.com/erlang-ls/erlang_ls/ to reflect the new variable.

Reply:
Also, thanks for the explicit ping on the PR. Is there any way to be automatically pinged if this file is modified?

Reply:
I guess you may subscribe to all notifications and filter only the notifications containing the filename.

Reply:
@yyoncho This sounds like a more powerful alternative: https://help.github.com/en/articles/about-code-owners

Reply:
@jfacorro I updated the README: https://github.com/erlang-ls/erlang_ls/commit/6477ec23b75e992f2dec5db39c8898be0bcc38d7